atd-vze/src/Components/DataTable.js

```diff
@@ -26,9 +35,18 @@ const DataTable = ({
     setEditField("");
   };
 
+  const onButtonClick = (e, section) => {
```

Comment:
This implementation looks great! One recommendation that I think is a good UX pattern for reverting changes or deleting data is prompting at confirm dialog box. Could we add that to this feature?

https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm